### DISCUSSION TITLE
RunTime too long!

### DISCUSSION THREAD

**Discussion by CompanionG** [openai/whisper#2164] [Show and tell]
i use model = whisper.load_model("base","cpu"), and other code is same with whisper examples,
![test](https://github.com/openai/whisper/assets/89848374/a6769b28-be56-4867-ba10-e79dbf575181)
5 seconds audio use 93.8684 seconds

but when i use model = whisper.load_model("base") 
same 5 second audio use 0.9159 seconds

i want to know whether the difference is normal?

**Comment by dragonfiber** [comment]
I believe it's because when you use the "cpu" parameter, you're putting the entire load of the computation on the cpu rather than the gpu.
